Task2/circle_area.cpp

```
@@ -0,0 +1,23 @@
+// Task 2
+// Write a function that will calculate the area of a circle
+// 
+#define _USE_MATH_DEFINES
+
+#include <math.h>
+#include <iostream>
+#include <assert.h>
+
+
+double circle_area(double radius) {
+	if (radius < 0) radius *= -1; // number area belong to N
```

I don't know the task description, but if the negative input is unspecified the function should report an error -throwing exception or whatever. Right now name of the function and output mismatched (you can't have a circle with the negative radi)